libs/api/libbpf_map.cpp

```diff
@@ -184,6 +159,31 @@ bpf_map__fd(const struct bpf_map* map)
     return map ? map->map_fd : libbpf_err(-EINVAL);
 }
 
+struct bpf_map*
+bpf_object__find_map_by_name(const struct bpf_object* obj, const char* name)
+{
+    struct bpf_map* pos;
+
+    bpf_object__for_each_map(pos, obj)
+    {
+        if (pos->name && !strcmp(pos->name, name))
```

Comment:
case insensitive compare?

Reply:
No, we want to have identical behavior to libbpf.  This code is directly copied from there.
(And in the future once libbpf is factored, this copy will be deleted.)